lumicks/pylake/kymo.py

```diff
@@ -570,6 +572,74 @@ def plot_with_time_histogram(
         ax_hist.set_ylabel("counts")
         ax_hist.set_title(self.name)
 
+    def bead_roi(self, bead_diameter, *, channel="green", plot=False, percentile=70, **kwargs):
```

Comment:
I think `bead_edges` sounds better. Additionally, we try to use verbs for method names so I would go with `estimate_bead_edges()`. This also makes it clear that it's an estimation and not like 100% bullet proof

Reply:
`estimate_bead_edges` it is!